lib/cli/interactive-setup/deploy.js

```diff
@@ -159,6 +159,10 @@ module.exports = {
       commands: ['deploy'],
       options: {},
     });
+    // This is a hack to disable local fallback for interactive setup
+    // After https://github.com/serverless/serverless/issues/1720 is addressed
+    // We can change the approach to deployment in interactive setup and remove this hack
+    serverless.isLocallyInstalled = true;
```

Comment:
Let's also remove it before `serverless.run()` (that way we apply this hack only to prevent local fallback)